library/groovy

```diff
@@ -1,42 +1,82 @@
+Tags: 2.4.11-jre7, 2.4-jre7, jre7
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jre7
 
-Tags: 2.4.10-jdk7-alpine, 2.4-jdk7-alpine, jdk7-alpine
-GitCommit: dbc61ddbc286b5847e8778747264e92bcd29c93d
+Tags: 2.4.11-jdk7-alpine, 2.4-jdk7-alpine, jdk7-alpine
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jdk7-alpine
 
-Tags: 2.4.10-jre7-alpine, 2.4-jre7-alpine, jre7-alpine
-GitCommit: dbc61ddbc286b5847e8778747264e92bcd29c93d
+Tags: 2.4.11-jre7-alpine, 2.4-jre7-alpine, jre7-alpine
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jre7-alpine
 
-Tags: 2.4.10-jdk8, 2.4-jdk8, jdk8, 2.4.10-jdk, 2.4-jdk, jdk
-GitCommit: dbc61ddbc286b5847e8778747264e92bcd29c93d
+Tags: 2.4.11-jdk8, 2.4-jdk8, jdk8, 2.4.11-jdk, 2.4-jdk, jdk
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jdk8
 
-Tags: 2.4.10-jre8, 2.4-jre8, jre8, 2.4.10-jre, 2.4-jre, jre, 2.4.10, 2.4, latest
-GitCommit: dbc61ddbc286b5847e8778747264e92bcd29c93d
+Tags: 2.4.11-jre8, 2.4-jre8, jre8, 2.4.11-jre, 2.4-jre, jre, 2.4.11, 2.4, latest
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jre8
 
-Tags: 2.4.10-jdk8-alpine, 2.4-jdk8-alpine, jdk8-alpine, 2.4.10-jdk-alpine, 2.4-jdk-alpine, jdk-alpine
-GitCommit: dbc61ddbc286b5847e8778747264e92bcd29c93d
+Tags: 2.4.11-jdk8-alpine, 2.4-jdk8-alpine, jdk8-alpine, 2.4.11-jdk-alpine, 2.4-jdk-alpine, jdk-alpine
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jdk8-alpine
 
-Tags: 2.4.10-jre8-alpine, 2.4-jre8-alpine, jre8-alpine, 2.4.10-jre-alpine, 2.4-jre-alpine, jre-alpine, 2.4.10-alpine, 2.4-alpine, alpine
-GitCommit: dbc61ddbc286b5847e8778747264e92bcd29c93d
+Tags: 2.4.11-jre8-alpine, 2.4-jre8-alpine, jre8-alpine, 2.4.11-jre-alpine, 2.4-jre-alpine, jre-alpine, 2.4.11-alpine, 2.4-alpine, alpine
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jre8-alpine
 
-Tags: 2.4.10-jdk9, 2.4-jdk9, jdk9
-GitCommit: dbc61ddbc286b5847e8778747264e92bcd29c93d
+Tags: 2.4.11-jdk9, 2.4-jdk9, jdk9
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jdk9
 
-Tags: 2.4.10-jre9, 2.4-jre9, jre9
-GitCommit: dbc61ddbc286b5847e8778747264e92bcd29c93d
+Tags: 2.4.11-jre9, 2.4-jre9, jre9
+GitCommit: 2ddaa9372c9d8c80fd76e0de808be030cd4ba380
 Directory: jre9
+
+
+Tags: 2.5.0-alpha-1-jdk7, 2.5-jdk7
+GitFetch: refs/heads/pre-release
+GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+Directory: jdk7
+
+Tags: 2.5.0-alpha-1-jre7, 2.5-jre7
+GitFetch: refs/heads/pre-release
+GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+Directory: jre7
+
+Tags: 2.5.0-alpha-1-jdk7-alpine, 2.5-jdk7-alpine
+GitFetch: refs/heads/pre-release
+GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+Directory: jdk7-alpine
+
+Tags: 2.5.0-alpha-1-jre7-alpine, 2.5-jre7-alpine
+GitFetch: refs/heads/pre-release
+GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+Directory: jre7-alpine
+
+Tags: 2.5.0-alpha-1-jdk8, 2.5-jdk8, jdk8, 2.5.0-alpha-1-jdk, 2.5-jdk
```

Comment:
Oops...was trying to bang out real quick before boarding my flight and missed a few places I copy/pasted from Java 8 images.  I'll fix.